charts/db-instances/values.yaml

```diff
@@ -2,158 +2,7 @@ dbinstances: {}
-          - numbackends:
-              usage: "COUNTER"
-              description: "Number of backends currently connected to this database."
-          - xact_commit:
-              usage: "COUNTER"
-              description: "Number of transactions in this database that have been committed"
-          - xact_rollback:
-              usage: "COUNTER"
-              description: "Number of transactions in this database that have been rolled back"
-          - blks_read:
-              usage: "GAUGE"
-              description: "Number of disk blocks read in this database"
-          - blks_hit:
-              usage: "GAUGE"
-              description: "Number of times disk blocks were found already in the buffer cache, so that a read was not necessary"
-          - tup_fetched:
-              usage: "COUNTER"
-              description: "Number of rows fetched by queries in this database"
-          - tup_inserted:
-              usage: "COUNTER"
-              description: "Number of rows inserted by queries in this database"
-          - tup_updated:
-              usage: "COUNTER"
-              description: "Number of rows updated by queries in this database"
-          - tup_deleted:
-              usage: "COUNTER"
-              description: "Number of rows deleted by queries in this database"
-          - temp_bytes:
-              usage: "GAUGE"
-              description: "Total amount of data written to temporary files by queries in this database."
-          - deadlocks:
-              usage: "COUNTER"
-              description: "Number of deadlocks detected in this database"
-      pg_stat_statements:
-        query: "SELECT userid, pgss.dbid, pgdb.datname, queryid, query, calls, total_time, mean_time, rows FROM pg_stat_statements pgss LEFT JOIN (select oid as dbid, datname from pg_database) as pgdb on pgdb.dbid = pgss.dbid WHERE not queryid isnull ORDER BY mean_time desc limit 20"
-        metrics:
-          - userid:
-              usage: "LABEL"
-              description: "User ID"
-          - dbid:
-              usage: "LABEL"
-              description: "database ID"
-          - datname:
-              usage: "LABEL"
-              description: "database NAME"
-          - queryid:
-              usage: "LABEL"
-              description: "Query unique Hash Code"
-          - query:
-              usage: "LABEL"
-              description: "Query class"
-          - calls:
-              usage: "COUNTER"
-              description: "Number of times executed"
-          - total_time:
-              usage: "COUNTER"
-              description: "Total time spent in the statement, in milliseconds"
-          - mean_time:
-              usage: "GAUGE"
-              description: "Mean time spent in the statement, in milliseconds"
-          - rows:
-              usage: "COUNTER"
-              description: "Total number of rows retrieved or affected by the statement"
+    image: prometheuscommunity/postgres-exporter:v0.15.0
```

Comment:
@dabde I'm not using the exporter atm, and hence I don't know what these queries were used for. Could you have a look and say whether we need them or not?

Reply:
FYI the queries have been moved to collectors in the upstream repository https://github.com/prometheus-community/postgres_exporter/issues/909. That's the reason they deprecated custom queries.

Reply:
@allanger looks fine. We didn't had any custom queries here ... only 1:1 was now under the collector defined.